### ISSUE TITLE
Provide facets to locate rows which generate certain quality assurance issues in the Wikibase extension

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#5033]
In the "Issues" tab, we report issues about Wikibase edits generated by the project grid and the schema. We give some examples of the values which generate the issues, but it would be much more convenient if users could simply get a facet which selects which rows generate each issue.

Not every issue can be tracked to a single row: for instance, the addition of two different identifier values on a given item is likely due to two different rows contributing those identifier values independently.

### Proposed solution

For issues which can be tracked back to a single row (such as string formatting issues), add a button on the issue to create a facet which selects the rows generating that issue. I propose the following internal architecture:
* the wikibase extension registers a new GREL function (for instance called `wikibaseIssues()`), whose task is to evaluate the schema on the current row, run scrutinizers on it, and return the list of issue identifiers that it generates
* the quality issues which are "facetable" are declared as such, and when they are, the UI displays a button to create the corresponding facet (for instance defined by `grel:wikibaseIssues().inArray('duplicate-whitespace')`)

### Alternatives considered

We could also try to extend this feature to issues which can be generated by multiple rows, but that is likely to require a more complicated architecture, so I would propose to leave that out first.

**Comment by ostephens** [comment]
This feels like it could be a useful pattern in other circumstances where it makes sense to validate a row to a schema or otherwise do quality checks on a row.

Do you think there is any way in which this could be implemented in an extensible way that could potentially then be used by multiple extensions, rather than simply a set of custom GREL/Facets in the Wikibase extension?

**Comment by wetneb** [comment]
Yes, I think we could go in that direction, but that would probably require first having a generic notion of quality assurance issues. This is something that would totally make sense and I thought we already had an issue about it, but I cannot find it at the moment.
